cfgov/jinja2/v1/index.html

```diff
@@ -5,8 +5,8 @@
 {%- endblock %}
 
 {% block desc -%}
-    Our vision is a consumer finance marketplace that works for American consumers,
-    responsible providers, and the economy as a whole.
+    {{ 'Our vision is a consumer finance marketplace that works for American consumers, ' +
+       'responsible providers, and the economy as a whole.' }}
```

Comment:
Yeah, I understand what your potential concern was, I was just advocating for the simpler solution of not breaking the line :)